.github/workflows/build-and-publish-images.yml

```diff
@@ -54,50 +58,30 @@ jobs:
           username: ${{ secrets.GPUCIBOT_DOCKERHUB_USER }}
           password: ${{ secrets.GPUCIBOT_DOCKERHUB_TOKEN }}
       - name: Create multiarch manifest
-        run: |
-          LATEST_CUDA_VER=$(yq '.CUDA_VER | sort | .[-1]' axis.yaml)
-          LATEST_PYTHON_VER=$(yq -o json '.PYTHON_VER' axis.yaml | jq -r 'max_by(split(".") | map(tonumber))')
-          LATEST_UBUNTU_VER=$(yq '.LINUX_VER | map(select(. == "*ubuntu*")) | sort | .[-1]' axis.yaml)
-
-          source_tags=()
-          tag="rapidsai/${{ matrix.IMAGE_REPO }}:cuda${{ matrix.CUDA_VER }}-${{ matrix.LINUX_VER }}-py${{ matrix.PYTHON_VER }}"
-          for arch in $(echo '${{ toJSON(matrix.ARCHES) }}' | jq .[] -r); do
-            source_tags+=("${tag}-${arch}")
-          done
+        env:
+          ARCHES: ${{ toJSON(matrix.ARCHES) }}
+          CUDA_VER: ${{ matrix.CUDA_VER }}
+          LINUX_VER: ${{ matrix.LINUX_VER }}
+          PYTHON_VER: ${{ matrix.PYTHON_VER }}
+          IMAGE_REPO: ${{ matrix.IMAGE_REPO }}
+          BUILD_TYPE: ${{ inputs.build_type }}
+          IMAGE_NAME: ${{ matrix.IMAGE_NAME }}
+        run: ci/create-multiarch-manifest.sh
 
-          docker manifest create ${tag} ${source_tags[@]}
-          docker manifest push ${tag}
-          if [[
-            "${LATEST_UBUNTU_VER}" == "${{ matrix.LINUX_VER }}" &&
-            "${LATEST_CUDA_VER}" == "${{ matrix.CUDA_VER }}" &&
-            "${LATEST_PYTHON_VER}" == "${{ matrix.PYTHON_VER }}"
-          ]]; then
-            docker manifest create rapidsai/${{ matrix.IMAGE_REPO }}:latest ${source_tags[@]}
-            docker manifest push rapidsai/${{ matrix.IMAGE_REPO }}:latest
-          fi
   delete-temp-images:
-    if: inputs.build_type == 'branch'
     needs: [compute-matrix, build-multiarch-manifest]
+    runs-on: ubuntu-latest
+    if: inputs.build_type == 'branch'
```

Comment:
```suggestion
```

This job looks like it only removes the arch-specific images, which means it's fine to run on PRs as well.